packages/salesforcedx-vscode-apex/src/commands/forceApexTestSuite.ts

```diff
@@ -0,0 +1,204 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { TestService } from '@salesforce/apex-node';
+import {
+  LibraryCommandletExecutor,
+  SfdxCommandlet,
+  SfdxWorkspaceChecker
+} from '@salesforce/salesforcedx-utils-vscode/out/src';
+import {
+  CancelResponse,
+  ContinueResponse,
+  ParametersGatherer
+} from '@salesforce/salesforcedx-utils-vscode/out/src/types';
+import { readFileSync } from 'fs';
+import { basename } from 'path';
+import * as vscode from 'vscode';
+import { OUTPUT_CHANNEL } from '../channels';
+import { workspaceContext } from '../context';
+import { nls } from '../messages';
+import {
+  ApexLibraryTestRunExecutor,
+  ApexTestQuickPickItem,
+  TestType
+} from './forceApexTestRun';
+
+export type ApexTestSuiteOptions = { suitename: string; tests: string[] };
+
+async function listApexClassItems(): Promise<ApexTestQuickPickItem[]> {
+  const apexClasses = await vscode.workspace.findFiles('**/*.cls');
+  const apexClassItems: ApexTestQuickPickItem[] = [];
+
+  apexClasses.forEach(apexClass => {
+    const fileContent = readFileSync(apexClass.fsPath).toString();
```

Comment:
We should keep in mind the perf impact of this. I tested in perf-project and it took ~1s for loading the menu.

Reply:
A change at this moment might not be needed, but if there's performance issues we should consider if we can reuse information from language server.

Reply:
I agree - I followed the pattern we're using [elsewhere](https://github.com/forcedotcom/salesforcedx-vscode/blob/5696483cc6e5dd71767312a9130ab8ddb1982fce/packages/salesforcedx-vscode-apex/src/commands/forceApexTestRun.ts#L82), but I think both should be examined to see if we can find a better approach.